airgun/session.py

```diff
@@ -159,10 +159,13 @@ def __init__(self, session_name=None, user=None, password=None):
             beused.
         :param str optional password: password for provided user. If None is
             passed - default one from settings will be used.
+        :param requests.sessions.Session optional session: session object to be used
+            to bypass login
```

Comment:
"session" is a bit loaded term, as it can mean Airgun session, web driver session (browser instance) and here - requests object of related HTTP calls. At the very least, modify comment to make it explicit that we expect requests.session object here.

But since `get_web_session()` helper is never used in Robottelo, maybe modify it to return `_session` cookie content, and just pass string here?

Reply:
+1 to session_cookie just to avoid too many sessions names.